packages/core/admin/admin/src/content-manager/components/ListViewTable/BulkActionButtons/tests/index.test.js

```diff
@@ -20,12 +22,19 @@ jest.mock('@strapi/helper-plugin', () => ({
 }));
 
 jest.mock('react-redux', () => ({
-  useSelector: () => ({
-    data: [
-      { id: 1, publishedAt: null },
-      { id: 2, publishedAt: '2023-01-01T10:10:10.408Z' },
-    ],
-  }),
+  useSelector() {
+    return {
+      data: [
+        { id: 1, publishedAt: null },
+        { id: 2, publishedAt: '2023-01-01T10:10:10.408Z' },
+      ],
+      contentType: {
+        settings: {
+          mainField: 'name',
+        },
+      },
+    };
+  },
 }));
```

Comment:
Can you explain why you're mocking a hook instead of just making a lighter store?